javascript-modal/styles.css

```diff
@@ -0,0 +1,73 @@
+* {
+  font-family: sans-serif;
+}
+
+.modal-button {
+  color: white;
+  background-color: rgb(22, 85, 202);
+  height: 50px;
+  width: 150px;
+  font-size: 17px;
+}
+
+.modal-two {
+  text-align: center;
+}
+
+.modal-one {
+  text-align: center;
+  height: 80px;
+}
+
+.no-button {
+  width: 98px;
+  height: 43px;
+  background-color: #f73434;
+  color: white;
+  font-size: 20px;
+  border-radius: 6px;
+}
+
+.modal {
+  position: fixed;
+  background: white;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
```

Comment:
Do you know how this is working? I would try to center this using tools I have taught you, such as `flexbox`

Reply:
<img width="1440" alt="Screen Shot 2021-04-04 at 1 25 19 PM" src="https://user-images.githubusercontent.com/69738204/113516668-8c10e200-9549-11eb-96f0-f8dd7c0db25f.png">
<img width="1440" alt="Screen Shot 2021-04-04 at 1 25 24 PM" src="https://user-images.githubusercontent.com/69738204/113516671-8e733c00-9549-11eb-9e0b-b8557fd39c69.png">